deltachat-ios/Controller/Settings/AdvancedViewController.swift

```diff
@@ -398,6 +398,10 @@ internal final class AdvancedViewController: UITableViewController {
             self.present(alert, animated: true, completion: nil)
         }))
 
+        alert.addAction(UIAlertAction(title: String.localized("learn_more"), style: .default, handler: { _ in
+            self.navigationController?.pushViewController(HelpViewController(dcContext: self.dcContext, fragment: "#importkey"), animated: true)
```

Comment:
What we could do prevent a retain cycle is to use a `[weak self]` and some `guard let` here. I don't consider this a blocker, just had to think about #2235

Reply:
i did a commit, thanks for the pointer.

it seems wise in general to prefer weak self for "back pointers" if in doubt, for the other occurances, however, that should go to a separate PR